doc/howtos/rdtraining/basicmodel.rst

```diff
@@ -0,0 +1,294 @@
+        postcode           | character varying           |           |          |
+        date_availability  | date                        |           |          |
+        expected_price     | double precision            |           |          |
+        selling_price      | double precision            |           |          |
+        bedrooms           | integer                     |           |          |
+        living_area        | integer                     |           |          |
+        facades            | integer                     |           |          |
+        garage             | boolean                     |           |          |
+        garden             | boolean                     |           |          |
+        garden_area        | integer                     |           |          |
+        garden_orientation | character varying           |           |          |
+        Indexes:
+            "estate_property_pkey" PRIMARY KEY, btree (id)
+        Foreign-key constraints:
+            "estate_property_create_uid_fkey" FOREIGN KEY (create_uid) REFERENCES res_users(id) ON DELETE SET NULL
+            "estate_property_write_uid_fkey" FOREIGN KEY (write_uid) REFERENCES res_users(id) ON DELETE SET NULL
+
+
+There are two broad categories of fields: 'simple' fields which are atomic
+values stored directly in the model's table and 'relational' fields linking
+records (of the same model or of different models).
+
+Example of simple fields are :class:`~odoo.fields.Boolean`,
+:class:`~odoo.fields.Float`, :class:`~odoo.fields.Char`,
+:class:`~odoo.fields.Date` or :class:`~odoo.fields.Selection`.
+
+.. exercise:: Add basic fields to the Real Estate Property table
+
+    Add the following basic fields to the table:
+
+    ========================= =========================
+    Field                     Type
+    ========================= =========================
+    name                      Char
+    description               Text
+    postcode                  Char
+    date_availability         Date
+    expected_price            Float
+    selling_price             Float
+    bedrooms                  Integer
+    living_area               Integer
+    facades                   Integer
+    garage                    Boolean
+    garden                    Boolean
+    garden_area               Integer
+    garden_orientation        Selection
+    ========================= =========================
+
+    The ``garden_orientation`` fields must have 4 options: 'North', 'South', 'East' and 'West'. The
+    selection list is defined as a list of tuples, see
+    `here <https://github.com/odoo/odoo/blob/b0e0035b585f976e912e97e7f95f66b525bc8e43/addons/crm/report/crm_activity_report.py#L31-L34>`__
+    for example.
+
+When the fields are added to the model, restart the server with ``-u estate``
+
+.. code-block:: console
+
+    $ ./odoo-bin --addons-path=../custom,../enterprise/,addons -d rd-demo -u estate
```

Comment:
Don't know if it's a good idea, but mention that if an error ok kind : NameError: name 'fields' is not defined, it's because we forgot to put it in the import ?

Reply:
I'd keep it this way since it is a standard Python error. I think the trainee should be able to solve it on his own.